Sources/OutlineView/OutlineView.swift

```diff
@@ -59,23 +176,68 @@ where Data.Element: Identifiable {
     ///     element in `data`. An `NSTableCellView` subclass is preferred.
     ///     The `NSView` should return the correct `fittingSize`
     ///     as it is used to determine the height of the cell.
-    public init(
+    init(
         _ data: Data,
         children: KeyPath<Data.Element, Data?>,
         selection: Binding<Data.Element?>,
         content: @escaping (Data.Element) -> NSView
     ) {
         self.data = data
-        self.children = children
+        self.childSource = .keyPath(children)
         self._selection = selection
         self.separatorVisibility = .hidden
         self.content = content
     }
 
     /// Creates an outline view from a collection of root data elements and
-    /// a key path to its children. The outline view will have row separator
-    /// lines enabled by default, the insets to which will determined by
-    /// the `separatorInsets` closure.
+    /// a closure that provides children to each element.
+    ///
+    /// This initializer creates an instance that uniquely identifies views
+    /// across updates based on the identity of the underlying data element.
+    ///
+    /// All generated rows begin in the collapsed state.
+    ///
+    /// Make sure that the identifier of a data element only changes if you
+    /// mean to replace that element with a new element, one with a new
+    /// identity. If the ID of an element changes, then the content view
+    /// generated from that element will lose any current state and animations.
+    ///
+    /// - NOTE: All elements in data should be uniquely identified. Data with
+    /// elements that have a repeated identity are not supported.
+    ///
+    /// - Parameters:
+    ///   - data: A collection of tree-structured, identified data.
+    ///   - children: A closure whose non-`nil` return value gives the
+    ///     children of `data`. A non-`nil` but empty value denotes an element
+    ///     capable of having children that's currently childless, such as an
+    ///     empty directory in a file system. On the other hand, if the value
+    ///     from the closure is `nil`, then the outline group treats `data` as a
```

Comment:
This was not introduced by you, but can you replace "group" with "view" in all instances of this comment?

```suggestion
    ///     from the closure is `nil`, then the outline view treats `data` as a
```

Reply:
No problem! Fixed https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3